packages/core/src/protocol-helpers.ts

```diff
@@ -360,14 +386,17 @@ export default class ProtocolHelpers {
         if (interaction.forms !== undefined && finalFormIndex === -1) {
             if (operationName !== undefined) {
```

Comment:
Maybe this allows for getting rid of the comments below?

```suggestion
        if (interaction.forms != null && finalFormIndex === -1) {
            if (operationName != null) {
```